sphinx_gallery/binder.py

```diff
@@ -158,3 +227,15 @@ def check_binder_conf(binder_conf):
             'for sphinx-gallery. A path to at least one of these files '
             'must exist in your Binder dependencies.')
     return binder_conf
+
+
+def _get_gallery_dir_path(fname, gallery_conf):
```

Comment:
I don't really understand why this function is needed. Also, I don't really understand what it is trying to do. Can you do a unit test of this and parametrize to many inputs? So that I can get an idea of what this function is trying to do and be sure we need it. I apologize not yet running the code and thus having little idea what goes on in the building of the binder directory.

Reply:
actually I think that I can accomplish the same thing with `relpath` now that I think about it, lemme give that a shot